<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bgd_wgt" native="true">
     <property name="styleSheet">
      <string notr="true">#bgd_wgt{border-image: url(:/login/images/background_regsiter.jpg);}</string>
     </property>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	border-radius:20px;
	background-color:rgb(0, 170, 127);
	color:white;
}
QPushButton:hover{

	border:2px solid rgb(0, 255, 255);
	border-radius:20px;
	
}
</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Menu_btn">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	border-radius:20px;
	background-color:rgb(0, 170, 127);
	color:white;
}
QPushButton:hover{

	border:2px solid rgb(0, 255, 255);
	border-radius:20px;
	
}
QPushButton:checked{

	background-color:rgb(63, 126, 31);
	
}</string>
      </property>
      <property name="text">
       <string>菜单</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	border-radius:20px;
	background-color:rgb(0, 170, 127);
	color:white;
}
QPushButton:hover{

	border:2px solid rgb(0, 255, 255);
	border-radius:20px;
	
}
</string>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="about_btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	border-radius:20px;
	background-color:rgb(0, 170, 127);
	color:white;
}
QPushButton:hover{

	border:2px solid rgb(0, 255, 255);
	border-radius:20px;
	
}
</string>
      </property>
      <property name="text">
       <string>关于</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>245</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="name_lb">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
color:rgb(220, 84, 71);
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>姓    名(&amp;u)：</string>
         </property>
         <property name="buddy">
          <cstring>name_le</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name_le">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
border-bottom:2px solid rgb(220, 84, 71);
background-color: transparent;
color:rgb(220, 84, 71);
font-size:12px;
font-weight:500;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pwd_lb">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
color:rgb(220, 84, 71);
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>密    码(&amp;p)：</string>
         </property>
         <property name="buddy">
          <cstring>pwd_le</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pwd_le">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
border-bottom:2px solid rgb(220, 84, 71);
background-color: transparent;
color:rgb(220, 84, 71);
font-size:12px;
font-weight:500;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cfm_lb">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
color:rgb(220, 84, 71);
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>确认密码(&amp;c)：</string>
         </property>
         <property name="buddy">
          <cstring>cfm_le</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="cfm_le">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
border-bottom:2px solid rgb(220, 84, 71);
background-color: transparent;
color:rgb(220, 84, 71);
font-size:12px;
font-weight:500;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="register_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color:rgb(220, 84, 71);
	color:white;
	border-radius: 10px;
 
}
QPushButton:disabled {
	background-color:rgb(150, 150, 150);
	color:rgb(188, 188, 188);
 
}
QPushButton:hover {
	background-color:rgb(255, 96, 82);
 
}
QPushButton:pressed {
	background-color:rgb(177, 67, 57);
 
}
</string>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>exit_btn</zorder>
     <zorder>reset_btn</zorder>
     <zorder>about_btn</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>Menu_btn</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../login_register_calculate_case.qrc"/>
 </resources>
 <connections/>
</ui>
